server.js

```diff
@@ -1,25 +1,68 @@
 const express = require('express');
-const webpackDevMiddleware = require('webpack-dev-middleware');
 const webpack = require('webpack');
-const webpackConfig = require('./webpack.config.js');
 const app = express();
+const bodyParser = require('body-parser');
+const mockMiddleware = require('./src/mock_endpoints/mockMiddleware');
+const path = require('path');
+require('dotenv').config();
 
-const compiler = webpack(webpackConfig);
+if (process.env.NODE_ENV === 'development') {
+	const webpackDevMiddleware = require('webpack-dev-middleware');
+	const webpackHotMiddleware = require('webpack-hot-middleware');
+	const webpackConfig = require('./webpack.dev.js');
+	const compiler = webpack(webpackConfig);
+	app.use(
+		webpackDevMiddleware(compiler, {
+			hot: true,
+			publicPath: webpackConfig.output.publicPath
+		})
+	);
+	app.use(webpackHotMiddleware(compiler));
+    app.use(express.static(path.join(__dirname, '/')));
+    app.use(bodyParser.json());
+    
+    app.get('/api', (req, res, next) => {
+        console.log('Should server mock data');
```

Comment:
is this console.log in use??

Reply:
That route is not in use, I have removed it